frontend/GameGUI.py

```diff
@@ -23,12 +24,13 @@ def check_events(self):
                 sys.exit()
 
     def run_game(self):
+        self.board.build_board() #does this need to be called constantly?
+    
         while True:
             self.check_events()
             pg.display.update()
             self.clock.tick(60)
-            self.board.build_board()
-            self.board.getCellClicked()
+            self.board.get_cell_clicked() #this runs too much. reduce or face errors
```

Comment:
I think this is a good point because when console logging the cell its clicked, it shows it was clicked like 4 or 5 times per click. Not sure what to do about it though. We need an investigation on it.

Reply:
Investigation issue logged https://github.com/Advanced-Software-Engineering-UMKC/Nine-Mens-Morris/issues/21

Reply:
Yeah, I don't know how to fix that, but I did think it was important to note. The number of times it runs depends entirely on the length of the click -- if you hold the left mouse button down, it'll go on infinitely, lol. Makes testing the piece placement pretty easy though, since you can just drag the mouse around the screen.